pandas/io/formats/printing.py

```diff
@@ -229,7 +229,7 @@ def as_escaped_string(
             max_seq_items=max_seq_items,
         )
     elif isinstance(thing, str) and quote_strings:
-        result = "'{thing}'".format(thing=as_escaped_string(thing))
+        result = f"'{as_escaped_string(thing)}'"
```

Comment:
is this the same as 
```suggestion
        result = repr(as_escaped_string(thing))
```

Reply:
Not sure because a pair of single quotes is needed